qiita_pet/templates/upload.html

```diff
@@ -8,7 +8,7 @@
       <br/><br/>
       Currently we can process (<b>{{extensions.replace(',',', ')}}</b>):
       <ul>
-        <li>fastq or fastq.gz (gziped fastq) for sequences. Note that zip files can not be processed</li>
+        <li>fastq or fastq.gz (gzipped fastq) for sequences. Note that .zip files can not be processed</li>
```

Comment:
According to the issue, you need to put `.zip` in quotes (`'.zip'`) and add a period at the end of the sentence `... can not be processed.`